packages/beacon-node/src/api/impl/beacon/pool/index.ts

```diff
@@ -153,10 +152,8 @@ export function getBeaconPoolApi({
         })
       );
 
-      if (errors.length > 1) {
-        throw Error("Multiple errors on submitPoolBlsToExecutionChange\n" + errors.map((e) => e.message).join("\n"));
-      } else if (errors.length === 1) {
-        throw errors[0];
+      if (errors.length > 0) {
+        throw new IndexedError("Some errors submitting BLS to execution change", errors);
```

Comment:
Should be plural to match other error messages, looks like in https://github.com/ethereum/beacon-APIs/pull/279 it was missed to change the operationId to plural as well when it was changed to a batch API

```suggestion
        throw new IndexedError("Some errors submitting BLS to execution changes", errors);
```